x/emissions/keeper/inference_synthesis/synth_palette_forecast_implied.go

```diff
@@ -39,26 +39,30 @@ func (p *SynthPalette) CalcForecastImpliedInferences() (map[Worker]*emissionstyp
 
 			weightSum := alloraMath.ZeroDec()                 // denominator in calculation of forecast-implied inferences
 			weightInferenceDotProduct := alloraMath.ZeroDec() // numerator in calculation of forecast-implied inferences
-			err := error(nil)
 
 			// Calculate the forecast-implied inferences I_ik
 			if p.AllInferersAreNew {
-				// If all inferers are new, take regular average of inferences
+				// If all inferers are new, calculate the median of the inferences
 				// This means that forecasters won't be able to influence the network inference when all inferers are new
-				// However this seeds losses for forecasters for future rounds
+				// However, this seeds losses for forecasters for future rounds
 
+				inferenceValues := make([]alloraMath.Dec, 0, len(sortedInferersInForecast))
 				for _, inferer := range sortedInferersInForecast {
 					if p.InferenceByWorker[inferer] != nil {
```

Comment:
this not the best way to test for inclusion, right? cuz it throws. consider changing to `if _, ok := ...; ok {...}`